app/src/main/java/gholzrib/arctouchchallenge/core/models/Movie.java

```diff
@@ -0,0 +1,118 @@
+package gholzrib.arctouchchallenge.core.models;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+
+/**
+ * Created by Gunther Ribak on 29/11/2016.
+ * For more information contact me
+ * through guntherhr@gmail.com
+ */
+
+public class Movie implements Serializable {
+
+    Integer id;
+    String title;
+    String original_title;
+    String overview;
+    String release_date;
+    String poster_path;
+    String backdrop_path;
```

Comment:
I need to use snake_case because of the Gson library. In order to convert the JSON into an model object, the variable names must be the same.